main.css

```diff
@@ -0,0 +1,151 @@
+body {
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size: 14px;
+  line-height: 1.428571429;
+  color: #333;
+  background-image:url('bg.jpg');
+  background-position: center; 
+}
+
+#logo {
+  text-align: center;
+}
+
+#content {
+  padding: 1em;
+}
+
+.box {
+  border: 1px solid #999;
+  background: #FCFCFC;
+  padding: 1em;
+  margin-bottom: 0.5em;
+}
+
+.welcome {
+  text-align: center;
+}
+
+h1  {text-align: center;}
+h2  {text-align: center;
+     font-size: 20px;
+    font-weight: bold;}
+
+h4 {text-align: center;}
+h3 {
+  font-size: 2em;
+  text-align: center;
+    background-color: #F4EBC3;
+    
+    
+    
+}
+
+.error {
+  text-align: center;
+}
+
+.error h2 {
+  font-size: 1.5em;
+}
+
```

Comment:
Try to apply your whitespace and indentation consistently. A consistent approach helps other developers scan and understand your code at a quick glance.